client_lib/pulp/client/commands/repo/sync_publish.py

```diff
@@ -17,6 +19,14 @@
 DESC_PUBLISH_RUN = _('triggers an immediate publish of a repository')
 DESC_PUBLISH_STATUS = _('displays the status of a repository\'s publish tasks')
 
+DESC_DOWNLOAD = _('triggers an immediate download of a repository that is using '
```

Comment:
to emphasize that it goes in the task queue with everything else, I'd adjust this to "queues a full download of all units in a repository..."